app/javascript/pages/ListingDirectory/DirectoryHelpers.tsx

```diff
@@ -190,45 +191,61 @@ export const getTableSubHeader = (listing: RailsRentalListing) => {
   return null
 }
 
+// Get imageCardProps for a given listing
+const getImageCardProps = (listing, hasFiltersSet?: boolean) => ({
+  imageUrl: listing.imageURL,
+  subtitle: `${listing.Building_Street_Address}, ${listing.Building_City} ${listing.Building_State}, ${listing.Building_Zip_Code}`,
```

Comment:
![Screen Shot 2022-01-25 at 11 26 27 AM](https://user-images.githubusercontent.com/3513225/151045602-c0fbdfa6-0d9c-4abf-8407-46d94a259bf2.png)
any ideas for a less technical message we could fallback on in cases like this?

Reply:
there is also a ticket for that! DAH-1008

Reply:
good-first-ticket!!